Adyen.xcodeproj/project.pbxproj

```diff
@@ -2322,6 +2322,7 @@
 		E278EB3C22AA5E8700497FD5 /* Payment Methods */ = {
 			isa = PBXGroup;
 			children = (
+				F92CBA082812CAE300367820 /* Brands */,
```

Comment:
Not sure about `Brands` folder inside of `Payment Methods`.. 
It is not really a PaymentMethod + what else could go in this folder? 🤔

Maybe we can move `CardType.swift` to `AdyenCore/Core/Model` ?